doc/usage/Getting_Started.md

```diff
@@ -0,0 +1,475 @@
+The following is a summary of the available elements:
+
+* Buses:
+    * `Bus`: An electrical bus.
+
+* Branches:
+    * `Line`: A line connects two buses. The parameters of the line are defined by a `LineParameters` object.
+    * `LineParameters`: This object defines the parameters of a line  (model, impedance, etc.)
+    * `Switch`: A basic switch element.
+    * `Transformer`: A generic transformer. The parameters of the transformer are defined by a `TransformerParameters`
+      object.
+    * `TransformerParameters`: This object defines the parameters of a transformer (model, windings, etc.)
+
+* Loads:
+  The ZIP load model is available via the following classes:
+    * `ImpedanceLoad`: A constant impedance (Z) load: $S = |V|^2 \times \overline{Z}$, $|S|$ is proportional to $|V|^2$.
+    * `CurrentLoad` A constant current (I) load: $S = V \times \overline{I}$, $|S|$ is proportional to $|V|^1$.
+    * `PowerLoad`: A constant power (P) load: $S = \mathrm{constant}$, $|S|$ is proportional to $|V|^0$.
+
+  A power load can be made flexible (controllable) by using the following class:
+    * `FlexibleParameter`: This object defines the parameters of the flexible load's control (Maximum power, projection,
+      type, etc.)
+
+  Note that flexible loads are an advanced feature that most users don't need. They are explained in details [here](usage-flexible-loads).
+
+* Sources:
+    * `VoltageSource`: Represents an infinite power source with a constant voltage.
+
+* Others:
+    * `Ground`: A ground acts as a perfect conductor. If two elements are connected to the ground, the potentials at the
+      connection points are always equal.
+    * `PotentialRef`: A potential reference sets the reference of potentials in the network. It can be connected to
+      buses or grounds.
+
+For a more detailed description of the elements, please refer to the [API reference][api-models].
+
+Let's use some of these elements to build the following network with a voltage source, a simple
+line and a constant power load. This network is a low voltage network (three-phase + neutral wire).
+
+![Network](../_static/Getting_Started_Tutorial.svg)
+
+It leads to the following code
+
+```pycon
+>>> import numpy as np
+... from roseau.load_flow import *
+
+>>> # Create two buses
+... source_bus = Bus(id="sb", phases="abcn")
+... load_bus = Bus(id="lb", phases="abcn")
+
+>>> # Define the reference of potentials to be the neutral of the source bus
+... ground = Ground(id="gnd")
+... # Fix the potential of the ground at 0 V
+... pref = PotentialRef(id="pref", element=ground)
+... ground.connect(source_bus, phase="n")
+
+>>> # Create a LV source at the first bus
+... # Volts (phase-to-neutral because the source is connected to the neutral)
+... un = 400 / np.sqrt(3)
+... source_voltages = [un, un * np.exp(-2j * np.pi / 3), un * np.exp(2j * np.pi / 3)]
+... vs = VoltageSource(id="vs", bus=source_bus, phases="abcn", voltages=source_voltages)
```

Comment:
For all the voltage sources, lines and loads, the phases have been specified which is not mandatory. IMO the phases should not be specified in those cases, as there is more chance to do something wrong (for exemple adding an MV line with `"abc"` in a LV network `"abcn"`) without noticing. If the phases are not specified the user will get an error. The phases for those elements should only be done for special cases.